### ISSUE TITLE
Your project is not spartanized

### ISSUE THREAD

**Issue by yossigil** [TechnionYP5777/Info-Evaluation#109]
You **cannot** fix it in this semester.

![image](https://cloud.githubusercontent.com/assets/12775400/22335796/b7e688f6-e3e8-11e6-93ed-d64683dd3f94.png)

@idabran @TomPalny @yh-github @AdiOmari @orimarco

**Comment by SlasherTech** [comment]
I checked with all the group members and they told me they all spartanized their code before committing.
I also met with them and spartanizer shows no tips for any of us.
We checked that we have the latest version of spartanizer.
We really are not sure why do it shows such tips in the attached picture.

Is it possible to meet next semester to clarify this issue ? It is not the first time it happens and we always make sure that our code is spartanized . Therefore, I see no reason it always shows that the code is not spartanized.
Thank you very much.

@ward-mattar @geniashand @NetanelFelcher @osherh @viviansh
